deploy/kubernetes/deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       serviceAccount: csi-resizer
       containers:
         - name: csi-resizer
-          image: gcr.io/k8s-staging-sig-storage/csi-resizer:canary
+          image: registry.k8s.io/k8s-staging-sig-storage/csi-resizer:canary
```

Comment:
This image doesn't exist in the registry:

```
 % docker pull registry.k8s.io/k8s-staging-sig-storage/csi-resizer:canary
Error response from daemon: manifest for registry.k8s.io/k8s-staging-sig-storage/csi-resizer:canary not found: manifest unknown: Failed to fetch "canary" from request "/v2/k8s-staging-sig-storage/csi-resizer/manifests/canary".
```

Reply:
So what should be the image name

Reply:
The latest resizer image exists:

```
registry.k8s.io/sig-storage/csi-resizer:v1.5.0
```

Reply:
iic, the `canary` tag was not available in the previous repo either. so it is missing in the new redirected path.

Reply:
So should I use v1.5.1 image name?